iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -194,15 +194,16 @@ static bool isDispatchableOp(Operation *op) {
     return false;
   }
   // Only linalg ops are marked dispatchable.
-  if (op->getDialect() !=
-      op->getContext()->getLoadedDialect<linalg::LinalgDialect>()) {
+  if ((op->getDialect() !=
+       op->getContext()->getLoadedDialect<linalg::LinalgDialect>()) &&
+      !isa<SubTensorOp>(op)) {
     return false;
   }
   return !isAlwaysClonedIntoDispatchOp(op);
 }
 
 static bool isAlwaysFusedIntoDispatchOp(Operation *op) {
-  return isDispatchableOp(op) && isa<linalg::TensorReshapeOp>(op);
+  return isDispatchableOp(op) && isa<linalg::TensorReshapeOp, SubTensorOp>(op);
```

Comment:
Just to help myself understand...
Does that means its going to be fused with its producer ? e.g what will happen in that case? 
```mlir
%0 = linalg.matmul ...
%1 = subtensor %0 ...
```
They should in the same dispatch workgroup right ?

Reply:
No, we are mostly fusing with producer. So in this case if %1 is the output, then they will go into two separate dispatch regions. We can revisit if it becomes a performance issue.

Reply:
Thanks!